WalletWasabi.Backend/WalletWasabi.Backend.csproj

```diff
@@ -11,6 +11,8 @@
     <PackageTags>walletwasabi, wasabiwallet, wasabi, hiddenwallet, wallet, bitcoin, hbitcoin, nbitcoin, magicalcryptowallet, magicalwallet, tor, chaum, chaumian, zerolink, coinjoin, fungibility, privacy, anonymity</PackageTags>
     <RepositoryType>Git</RepositoryType>
     <RepositoryUrl>https://github.com/zkSNACKs/WalletWasabi/</RepositoryUrl>
+    <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
```

Comment:
If this set to true, it will always generate the lock file if there were updates? 

https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#enabling-lock-file

Reply:
I would rather say: It rewrites package.lock.json file when you modify dependencies. 

So it is easy to spot any dependence changes in git when you see that package.lock.json is modified.

Also, it's good for transitive dependencies (dependencies of dependencies) which you do not specify explicitly but you don't want them to change under your hands.